src/pages/Product/Product.js

```diff
@@ -0,0 +1,335 @@
+            <div>30평대</div>
+            <div>이틀 전</div>
+          </ProductHeader>
+          <ProductImgContainer>
+            <ProductImg
+              src="https://images.unsplash.com/photo-1501924497965-792fefaea3dc?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1489&q=80"
+              alt="product"
+            />
+          </ProductImgContainer>
+          <div></div>
+        </div>
+        <div>
+          <CommentCount>
+            댓글 <CommentCountNum>{commentList.length}</CommentCountNum>
+          </CommentCount>
+          <InputContainer>
+            <FaSmile className="fas fa-smile"></FaSmile>
+            <form onSubmit={onSubmitComment}>
+              <CommentInput
+                type="text"
+                name="comment"
+                placeholder="칭찬과 격려의 댓글은 작성자에게 큰 힘이 됩니다 :)"
+                value={inputValue}
+                onChange={onChangeCommentInput}
+              />
+              <Register alert={opacity}>등록</Register>
+            </form>
+          </InputContainer>
+          {commentList &&
+            commentList.map(comment => {
+              return (
+                <Comment key={comment.id}>
+                  <Circle alt="user face" src={comment.img_url} />
+                  <CommentFeed>
+                    <UserAndContent>
+                      <User>{comment.user}</User>
+                      <div>{comment.content}</div>
+                    </UserAndContent>
+                    <CommentInfo>
+                      <Time>{comment.date}</Time>
+                      <CommentHeart onClick={onClickCommentHeart}>
+                        <FaHeart className="far fa-heart"></FaHeart>
+                        <span>{comment.like}</span>
+                      </CommentHeart>
+                    </CommentInfo>
+                  </CommentFeed>
+                </Comment>
+              );
+            })}
+        </div>
+      </Article>
+      <Aside>
+        <div>
+          <LikeButton>
+            <FaHeart className="far fa-heart"></FaHeart>22
+          </LikeButton>
+          <Profile>
+            <UserInfo>
+              <Circle
+                alt="user face"
+                src="https://images.unsplash.com/photo-1506748686214-e9df14d4d9d0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=634&q=80"
+              />
+              <Name>jiyon</Name>
+            </UserInfo>
+            <Follow>팔로우</Follow>
+          </Profile>
+        </div>
+        <Circle as="div" onClick={onClickURL}>
+          URL
+        </Circle>
+      </Aside>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  display: flex;
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+`;
+
+const Article = styled.div`
+  padding: 20px;
+  border-right: 1px solid ${props => props.theme.borderLine};
+`;
+
+const ProductHeader = styled.div`
+  ${flexSet('space-between', 'center')}
```

Comment:
👍

Reply:
어제 본 바로는 align은 default value가 'center'로 되어있는 것 같아서 생략가능해보입니다!
```suggestion
  ${flexSet('space-between')}
```